frontend/app/auth/register/page.tsx

```diff
@@ -32,18 +36,33 @@ export default function Register() {
     email: '',
     password: '',
   })
-
+  const [isSubmitting, SetIsSubmitting] = useState(false)
   const [isChecked, setIsChecked] = useState(false)
+  const [errors, setErrors] = useState<ZodIssue[]>([])
+
+  const toast = useToast()
 
-  const handleSubmit = async (
-    e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+  const cretateAccount = async (
```

Comment:
関数名なのに
いかにも作られたアカウントが入ってくるような名前に変えたんでしょう？
理由があれば知りたいです！！

```suggestion
  const handleSubmit = async (
```

Reply:
@issiyda 
特に変更の理由はなく、これでいいかという感じで適当に付けてしまいました。すみません
handleSubmitに戻したので確認お願いします